sdk/test/Services/S3/IntegrationTests/CopyObjectTests.cs

```diff
@@ -157,12 +157,12 @@ public void TestCopyObjectWithTagsReplace()
             Assert.IsTrue(taggingMetadata.Tagging.Any(tag => tag.Key == "newtag1" && tag.Value == "1"));
         }
 
-        [DataRow(false, testKey, "/destinationTestKey1.txt", "destinationTestKey1.txt")]
-        [DataRow(true, testKeyWithSlash, "/destinationTestKey2.txt", "/destinationTestKey2.txt")]
-        [DataRow(true, testKeyWithSlash, "/", "/")]
+        [DataRow(testKey, "/destinationTestKey1.txt", "/destinationTestKey1.txt")]
```

Comment:
nit: I think it is better to remove the slash from the `destinationKey` instead of adding it to the `expectedKey` to make this test case different than the second one, or remove the slashes from the next test case.

Reply:
done

Reply:
I think this test could be even better if the data row was:
```
[DataRow(testKeyWithSlash, "destinationTestKey1.txt", "destinationTestKey1.txt")]
```

There's already another test case in this class copying `testKey` somewhere else. The code above will validate the leading slash is handled properly.